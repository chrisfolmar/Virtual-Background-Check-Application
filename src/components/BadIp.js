import React, { Component } from 'react'

class BadIp extends Component {

	 render(){

	 	const address = this.props.address;

        const redStyle = {
              color: 'red'
            };

		return(
	            <div className="features-icons-item mx-auto mb-5 mb-lg-0 mb-lg-3">
	                  <h4>{address}</h4>
	                  <h5 style={redStyle}>WARNING!</h5>
	                  <p className="mb-0">The IP was found in at least one blacklist!</p>
                </div>
			);
	}
}

export default BadIp;