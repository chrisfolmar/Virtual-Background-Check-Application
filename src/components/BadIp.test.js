import React from 'react';
import ReactDOM from 'react-dom';
import BadIp from './BadIp';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<BadIp />, div);
  ReactDOM.unmountComponentAtNode(div);
});