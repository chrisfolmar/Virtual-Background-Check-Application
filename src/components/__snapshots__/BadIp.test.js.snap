// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BadIp full snapshot rendering 1`] = `
<BadIp>
  <div
    className="features-icons-item mx-auto mb-5 mb-lg-0 mb-lg-3"
  >
    <h4 />
    <div
      className="features-icons-icon d-flex"
    >
      <i
        className="fa fa-ban m-auto"
        style={
          Object {
            "color": "red",
          }
        }
      />
    </div>
    <h5
      style={
        Object {
          "color": "red",
        }
      }
    >
      WARNING!
    </h5>
    <p
      className="mb-0"
    >
      The IP was found in at least one blacklist!
    </p>
  </div>
</BadIp>
`;
